src/main/java/th/ac/kmitl/science/comsci/example/nameprinters/EarlyNamePrinter.java

```diff
@@ -0,0 +1,25 @@
+package th.ac.kmitl.science.comsci.example.nameprinters;
+
+import java.io.PrintWriter;
+
+public class EarlyNamePrinter implements NamePrinter {
+    @Override
```

Comment:
please enter one line between class name and first method

Reply:
EarlyNamePrinter <- seems to be obvious. What does it mean ?

P.S. Meanings always matter for project with numbers of maintainers.